src/schema/types/boolean-type.ts

```diff
@@ -0,0 +1,17 @@
+import { CoreType, CoreTypeOptions } from './core-type';
+
+export class BooleanType extends CoreType {
+  constructor(name: string, options?: CoreTypeOptions) {
+    super(name, options, 'Boolean');
+  }
+
+  applyValidations(value: boolean): string[] {
```

Comment:
parameter `value` not used, returning empty array? I assume this will be updated?

Reply:
Fixed: a temporal validation was added, but this file will be updated in the current sprint (Sprint 3)